src/OrchardCore/OrchardCore.Localization.Abstractions/LocalizationOptionsUpdater.cs

```diff
@@ -12,11 +13,20 @@ namespace OrchardCore.Localization;
 /// This is mainly used in the localization module to update the current <see cref="RequestLocalizationOptions"/> that
 /// might set from other modules in Orchard Core pipeline.
 /// </remarks>
+[Obsolete("This class is deprecated, and will be removed in the upcoming major release.")]
 public class LocalizationOptionsUpdater
 {
-    private readonly bool _useUserOverride;
     private readonly RequestLocalizationOptions _options;
 
+    /// <summary>
+    /// Initializes a new instance of a <see cref="LocalizationOptionsUpdater"/>.
+    /// </summary>
+    /// <param name="options">The <see cref="RequestLocalizationOptions"/>.</param>
+    public LocalizationOptionsUpdater(RequestLocalizationOptions options)
```

Comment:
Seems there's no need for the `LocalizationOptionsUpdater` changes, right? I did it before I rid off the use of `LocalizationOptionsUpdater`